<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>775</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>75</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Программа численно решает
задачу Коши для ДУ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>115</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
<<<<<<< HEAD
     <pixmap>:/Resources/Diffeq.png</pixmap>
=======
     <pixmap resource="Resources.qrc">:/Resources/Diffeq.png</pixmap>
>>>>>>> a7a0e1b9d63f62df418160488f152a794e46c4b2
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_u0">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>271</y>
      <width>28</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Resources/u_0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_h">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>355</y>
      <width>22</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Resources/h.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_h">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_E_x">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>336</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Resources/x_eps.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_max">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>310</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
<<<<<<< HEAD
     <pixmap>:/Resources/x_max.png</pixmap>
=======
     <pixmap resource="Resources.qrc">:/Resources/x_max.png</pixmap>
>>>>>>> a7a0e1b9d63f62df418160488f152a794e46c4b2
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="Table">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>771</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_max">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_E_x">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>335</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_E_nu">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>385</y>
      <width>27</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Resources/v_eps.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_E_nu">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>385</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_a">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>207</y>
      <width>25</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Resources/a.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_a">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>205</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_teta">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>228</y>
      <width>22</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Resources/teta.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_teta">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>446</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Решить!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Результаты</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="mainPlot" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>771</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>350</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Справка по задаче</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tableButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>45</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Справка по таблице</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_N_max">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>407</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
<<<<<<< HEAD
     <pixmap>:/Resources/Nmax.png</pixmap>
=======
     <pixmap resource="Resources.qrc">:/Resources/Nmax.png</pixmap>
>>>>>>> a7a0e1b9d63f62df418160488f152a794e46c4b2
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
<<<<<<< HEAD
      <x>23</x>
      <y>510</y>
      <width>241</width>
      <height>131</height>
=======
      <x>20</x>
      <y>520</y>
      <width>211</width>
      <height>91</height>
>>>>>>> a7a0e1b9d63f62df418160488f152a794e46c4b2
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
<<<<<<< HEAD
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Пояснение к графикам:&lt;/span&gt;&lt;br/&gt;Красный - u(x) - точное решение ;&lt;br/&gt;Синий - v(x) - численное решение;&lt;br/&gt;Зелёный - v2(x) - численное решение, &lt;br/&gt;полученное с половинным шагом.&lt;br/&gt;Графики решений подписаны в формате:&lt;br/&gt;&lt;span style=&quot; color:#55007f;&quot;&gt;(a, ϑ, u0).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
=======
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Пояснение к графикам:&lt;/span&gt;&lt;br/&gt;Красный - u(x) - точное решение ;&lt;br/&gt;Синий - v(x) - численное решение;&lt;br/&gt;Зелёный - v2(x) - численное решение, &lt;br/&gt;полученное с половинным шагом.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
>>>>>>> a7a0e1b9d63f62df418160488f152a794e46c4b2
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
